### DISCUSSION TITLE
Videos only download 5%-10% (5-10 Minuites), Does not download the whole video.

### DISCUSSION THREAD

**Discussion by RedMfBackpack** [aclap-dev/video-downloadhelper#1405] [Help and Q&A]
### Your question

Hi. I just bought  the license key and my videos are still being capped at 5%-10%, (5-10 Minutes) and does not download the whole video :(. I am using Windows 11 and using Google Chrome. Can you guys please help me.

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"31274d7eb03a42b1"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "inbrowser",
    "used_history_button": true,
    "download_directory": "C:\\Users\\Trey\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\Trey\\Downloads"
        }
    },
    "concurrent_downloads_max": 10,
    "never_show_no_incognito_msg_again": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 14,
    "successfull_dl": 9,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by Mattmjs** [comment]
Yes can you provide something where this problem happens for others to look at.

**Reply by RedMfBackpack** [reply]
You mean the wesbites? I am trying to download from this wesbite. https://ww17.0123movie.net/. Or formally known as 123movies.

**Reply by Mattmjs** [reply]
I'm downloading something and 100% complete. Do you see a button that says list server, this gives you different options
to download from.

**Reply by Mattmjs** [reply]
Your on the latest extension version too, Firefox hasn't got yet.

**Reply by RedMfBackpack** [reply]
For some reason, I can download some videos and not others. Sometimes I can get downloads to get too 5%-10% all the way up to 50%, but they autocpmplete and its not the whole movie. I have no VPN on. I don't know why it isnt working right. I've tried doing the servers, all of them for a specific movie. Here is the movie if you'd like to try it. I will be surprised if you can download it fully. https://ww17.0123movie.net/movie/deadpool-wolverine-1630857355.html

**Reply by Mattmjs** [reply]
It's made it to 50% and still progressing, I'll see if it completes shortly. 100% downloaded.

**Reply by RedMfBackpack** [reply]
Well, Im not for sure whats going on then. Is there anyway I can send you anything to try to see what the problem is?

**Reply by Mattmjs** [reply]
My thought is your internet, if it's not delivering a consistent speed or perhaps it's drops right down.
If the speed stalls enough the download could possibly terminate early. Look at your network in the task manger.
Click processes and look at the column for network, it should show the speed for chrome as you have a download running.

**Reply by RedMfBackpack** [reply]
My internet speeds are decent. When you're downloading something, are you getting a instant download if you have high internet speeds? On the download bar, it says its going about 6-10mbps. My internet speeds when normally tested are 500mbps. I dont think anything is capping it, however I may be wrong.

**Reply by Mattmjs** [reply]
Looks like you have super fast internet, way faster than most people use including mine. Some sites however can throttle the 
download speed. Maybe try on Firefox like I did and see if it's different to chrome. Also possible is the connection momentarily
disconnects for a few seconds before reconnecting, I've seen that on my computer. Other than that don't know what else the issue could be.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

I'm going to guess at a couple of things here.  Typically, an ISP will tell you you're getting a 500M service.  What they don't tell you is that that is a 500 megaBIT service.  BIT.  Not byte.  Bit.  This inflates the speed by a factor of 8, being the number of bits in a byte.  In addition, they are abusing the prefix mega.  Mega is actually 1024 squared, a number somewhat larger than one million.  But when they say 500Mega they really mean 500 million.  Less than what an informed computer user would assume.  Computers work in binary & mega is a power of 2.  So an alleged 500 megabit connection is really only 62.5 million bytes per second.

There are 2 places you should be monitoring VDH download speed: The VDH menu, the Resource Monitor.  Both of those report download speeds in bytes per second.  The Resource Monitor dispenses with kilo & mega & just gives you straight bytes per second.  VDH calculates mega but it does it properly, in multiples of 1024 squared.

I'm assuming your claim of 6-10mbps is what you read on the VDH menu.  That is a number of megabytes per second.  You should be comparing that with the 62.5 million bytes per second I just explained, not the 500M of smoke your ISP is blowing in your eyes.

If you got 6-10 megabytes per second download speed, that is quite good.  It does indicate, however, that the web site is throttling their service.  If they weren't throttling their service, you would see download speeds on the order of the 62.5 million bytes per second that I've been mentioning.  So the web site is throttling their service, although they are being generous about it.
